test/resource/en.js

```diff
@@ -0,0 +1,15 @@
+export default 
```

Comment:
There's already a `resource-en.ts`. Can that be used? Also, maybe we should consider shipping an english culture module by default as well?

Reply:
Oh, I must have missed it.

Reply:
Removed the resource file I added and used the existing `resource-en` module.